### DISCUSSION TITLE
fslightbox.js on cdnjs

### DISCUSSION THREAD

**Discussion by user53245** [cdnjs/cdnjs#14068] [General] [answered]
Hi!

An excellent plug and play library for viewing a large version of an image.

[https://github.com/banthagroup](https://github.com/banthagroup)

**Comment by MattIPv4** [comment, marked as answer]
To request a library be added to cdnjs, please fill out the [cdnjs Library Request](https://github.com/cdnjs/packages/issues/new?assignees=&labels=%F0%9F%8F%B7+Library+Request&template=library-request.yml&title=Add%3A+xxx) issue form on the cdnjs/packages repo.
